front/components/ConnectorPermissionsModal.tsx

```diff
@@ -126,7 +126,11 @@ export default function ConnectorPermissionsModal({
         }
 
         await mutate(
-          `/api/w/${owner.sId}/data_sources/${dataSource.name}/managed/permissions`
+          (key) =>
+            typeof key === "string" &&
```

Comment:
do we still need the key thing ?

Reply:
Oh yeah for parents ?

Reply:
yes we still need it. It is here to invalidate all kids of this url if we see it as a tree with each letter being a kids.